userspace/engine/rule_loader_collector.cpp

```diff
@@ -248,6 +248,20 @@ void rule_loader::collector::append(configuration& cfg, rule_info& info)
 		prev->cond += info.cond;
 	}
 
+	if (!info.output.empty())
+	{
+		prev->output += " ";
+		prev->output += info.output;
+	}
+
+	if (!info.tags.empty())
+	{
+		for (auto itr : info.tags)
+		{
+			prev->tags.insert(itr);
```

Comment:
Do we want to check for empty tags or other sensitization checks? Need to check on existing mechanisms more, I marked this PR as wip for the moment.